src/drivers/uavcan/actuators/esc.cpp

```diff
@@ -230,3 +230,20 @@ void UavcanEscController::orb_pub_timer_cb(const uavcan::TimerEvent &)
 		_esc_status_pub = orb_advertise(ORB_ID(esc_status), &_esc_status);
 	}
 }
+
+uint8_t UavcanEscController::check_escs_status()
+{
+	int esc_status_flags = 255;
```

Comment:
Can you only set the bits for the ones that are really online?

Reply:
Actually yes, I think in an early implementation I used this logic to fix something but after reviewing makes completely sense to re-invert the logic :)!